### ISSUE TITLE
Configure WIP Web pages using Jekyll

### ISSUE THREAD

**Issue by matthew-mizielinski** [WCRP-CMIP/WGCM_Infrastructure_Panel#14]
Following some experimentation I've worked out how to set up a Jekyll configuration to allow the extension of the current site. This issue will cover an initial pull request with a modified layout and guidance on how to develop this using Jekyll.

At 0dc7ae8 this repo has a minimal `_config.yml` configuration file, a set of markdown pages and documents. This will be extended with a new layout, based on the existing `minimal` Jekyll theme and I'll add a set of CMIP6 links for starters

**Comment by matthew-mizielinski** [comment]
Added information on the Wiki here on how to set up a conda environment to work with the content. Will push branch to the repo tomorrow

**Comment by matthew-mizielinski** [comment]
Following the issues noted by @durack1 in the comments in #15 with the deployment I've worked out what the issues was -- all the jekyll files need to be within the /docs folder as this is where the web service picks up everything. Without this the pages could not be rendered due to an un-resolvable link. 

I've had a tinker with my own sandbox site and worked the problem through. I've merged #17 and the deployed website looks good as far as I can see.
